cmd/rhobs/requests.go

```
@@ -0,0 +1,298 @@
+	}
+
+	clusterSelector, err := metav1.LabelSelectorAsSelector(&metav1.LabelSelector{MatchLabels: map[string]string{
+		clusterIdCdLabel: cluster.ID(),
+	}})
+	if err != nil {
+		return "", fmt.Errorf("failed to create label selector for cluster '%s': %v", cluster.ID(), err)
+	}
+
+	var clusterDeployments hivev1.ClusterDeploymentList
+
+	hiveClient.List(context.TODO(), &clusterDeployments, &client.ListOptions{LabelSelector: clusterSelector})
```

golangci-lint (2.11.4)

[error] Error return value of `hiveClient.List` is not checked

(errcheck)